src/dotnet-install.ps1

```diff
@@ -729,6 +747,8 @@ try {
 }
 catch {
     Say "Cannot download: $DownloadLink"
+	SafeRemoveFile -Path $ZipPath
```

Comment:
Both files used spaces, but the new changes add tabs. It would be nice to unify these.